src/Model/Domain/OpenTracking.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * Copyright (C) 2013 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+namespace Mailgun\Model\Domain;
+
+/**
+ * Represents a single Open Tracking setting for a domain tracking.
+ *
+ * @author Artem Bondarenko <artem@uartema.com>
+ */
+final class OpenTracking
+{
+    private $active;
+
+    public static function create(array $data): self
+    {
+        $model = new self();
+        $active = $data['active'] ?? null;
+
+        if (true === $active) {
+            $model->active = 'yes';
+        } elseif (false === $active) {
+            $model->active = 'no';
+        } else {
+            $model->active = $active;
+        }
```

Comment:
Three cases seems unnecessary, given this case doesn't allow the `htmlonly` according to the API Docs

```php
        $model = new self();
        $model->active = $active ? 'yes' : 'no';
```